examples/simple.js

```diff
@@ -49,8 +41,11 @@ for (let i = 0; i < 10; i++) {
 		// handle this with defaults
 		console.log('default goes here')
 	} else {
-
-		console.log(`selected experiment '${experiment.name}' for '${JSON.stringify(visitor)}'`)
+		console.log(
```

Comment:
again, this is horrifying, the old version is much more readable than this.

Reply:
I've seen worse...
Bear in mind that even if the line length exceeds the maximum line width, as long as the expressions in the template literal are simple variables, prettier will keep the line as a whole.
So, extracting `experiment.name` & `JSON.stringify(visitor)` to vars will prevent this travesty.